welcome/oke_about.adoc

```diff
@@ -264,7 +264,7 @@ The following table is a summary of the feature availability in {oke} and {produ
 | Vertical Pod Autoscaler | Included | Included | Vertical Pod Autoscaler
 | Cluster Monitoring (Prometheus) | Included | Included | Cluster Monitoring
 | Device Manager (for example, GPU) | Included | Included | N/A
-| Log Forwarding | Not Included | Included | Red Hat OpenShift Logging Operator
+| Log Forwarding | Included | Included | Red Hat OpenShift Logging Operator
 | Telemeter and Insights Connected Experience | Included | Included | N/A
 s| Feature s| {oke} s| {product-title} s| Operator name
```

Comment:
Perhaps outside the scope of this review, but is this being one continuous table with the headers repeated throughout intentional?